### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by emrekuecuek**
A trivial test written for page controller class. Its working and returned "OK" from terminal. (Tested by phpunit)

### CODE REVIEW

**Review by karakayasemi — CHANGES_REQUESTED** [review]
please configure Travis. Thanks

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  - which php
+  - more $(which pecl)
+  - which phpize
+  - bash tests/.travis/before_install.sh $APP_NAME $CORE_BRANCH $DB
+  - make
+  - make appstore
+  - cd ../core
+  - php occ app:enable $APP_NAME
+
+script:
```
Comment (emrekuecuek) [inline .travis.yml]: I dont actually know how to give exact test path and how to execute travis ci tool, so it would be nice to discuss this

**Review by karakayasemi — CHANGES_REQUESTED** [review]
No need to Oracle test. You can use backup codes app as an example

File: .travis.yml (hunk 1 above)
Comment (karakayasemi) [reply .travis.yml]: No more babysitting, just spend time a bit, try to find yourself.

**Comment by emrekuecuek** [comment]
Travis tests fail upon older php versions while installing image, php 5.6 and php 7 images can not install owncloud properly.

**Comment by karakayasemi** [comment]
Everything looks okay, I checked, ownCloud core masters min version is now 7.1(since 2 days ago).  That is why our test failing for 5.6 and 7, just remove them.

**Comment by emrekuecuek** [comment]
I updated travis file, all images build and run smoothly all tests for now.

**Review by karakayasemi — APPROVED** [review]

**Pull request merged**